get_download_link.sh

```diff
@@ -183,12 +185,14 @@ get_link(){
     fi
 
   elif [[ "${PRODUCT}" = "psmdb" && "${BUILD_ARCH}" = "x86_64" ]]; then
-    if [[ "${DISTRIBUTION}" = "ubuntu" ]]; then OPT="xenial";
-    elif [[ "${DISTRIBUTION}" = "centos" ]]; then OPT="centos6";
-    else OPT="${DISTRIBUTION}"; fi
+    if [[ "${VERSION}" = "3.4" ]]; then
+      if [[ "${DISTRIBUTION}" = "ubuntu" ]]; then OPT="-xenial";
+      elif [[ "${DISTRIBUTION}" = "centos" ]]; then OPT="-centos6";
+      else OPT="-${DISTRIBUTION}"; fi
+    else OPT=""; fi
     if [[ -z ${VERSION_FULL} ]]; then
       if [[ ${SOURCE} = 0 ]]; then
-        LINK=$(wget -qO- https://www.percona.com/downloads/percona-server-mongodb-${VERSION}/LATEST/binary/|grep -oE "percona-server-mongodb-${VERSION}\.[0-9]+-[0-9]+(\.[0-9]+)?-${OPT}-${BUILD_ARCH}\.tar\.gz"|head -n1)
+        LINK=$(wget -qO- https://www.percona.com/downloads/percona-server-mongodb-${VERSION}/LATEST/binary/|grep -oE "percona-server-mongodb-${VERSION}\.[0-9]+-[0-9]+(\.[0-9]+)?${OPT}-${BUILD_ARCH}(\.glibc2.17)?\.tar\.gz"|head -n1)
```

Comment:
Maybe it would be good to not specify glibc 2.17 version here directly, but let it work for any if possible, but it's just a general comment.

Reply:
Good thought, thank you! I have changed it to allow for any glibc version here. There are other points where glibc versions are specified, but those are sometimes used to construct the link or to differentiate between versions, so I have left those as is.